miso-dto/pom.xml

```diff
@@ -22,7 +22,7 @@
     </dependency>
     <dependency>
       <groupId>uk.ac.bbsrc.tgac.miso</groupId>
-      <artifactId>miso-service</artifactId>
+      <artifactId>sqlstore</artifactId>
```

Comment:
I've updated this to depend on core instead. The dtos just need access to the model objects.